app/assets/stylesheets/common/fonts.scss

```diff
@@ -0,0 +1,298 @@
+.font-helvetica {
```

Comment:
I think this file has been incorrectly included in the PR?

Reply:
Yaa, i've deleted the file from this PR in the latest commit 😅😄